hibernate-core/src/main/java/org/hibernate/tool/schema/internal/exec/ScriptTargetOutputToFile.java

```diff
@@ -72,7 +87,7 @@ public void release() {
 	}
 
 	@SuppressWarnings("ResultOfMethodCallIgnored")
-	static Writer toFileWriter( File file, String charsetName ) {
+	static Writer toFileWriter( File file, String charsetName, boolean append ) {
```

Comment:
As per our convention, we need space delimiter when we invoke some method but we don't need it for method declaration, right?

Reply:
Thanks @NathanQingyangXu , fixed it!